server/api/swagger.yaml

```diff
@@ -860,6 +860,38 @@ paths:
           description: The service
           schema:
             type: object
+  /services/{service}/health/{slice}:
+    x-swagger-router-controller: servicesController
+    get:
+      operationId: getServiceHealth
+      summary: Get a currently deployed service.
+      tags:
+        - Service
+      parameters:
+        - name: service
+          in: path
+          type: string
+          required: true
+        - name: environment
+          in: query
+          type: string
+          required: true
+        - name: serverRole
```

Comment:
If a user wanted information about the _blue_ slice of _some-service_ running on the ASG _c01-xx-SomeRole-blue_, would they submit _SomeRole_ or _SomeRole-blue_ as the _serverRole_ argument?